src/com/gr1t/taskmanager/Main.java

```diff
@@ -71,14 +74,50 @@ public static void main(String[] args) {
                     TaskManager.printAllTasks();
                     break;
                 case 5:
+                    TaskManager.sortByStatus();
+                    System.out.println("List after sorting by status: ");
+                    TaskManager.printAllTasks();
+                    break;
+                case 6:
+                    TaskManager.printAllTasks();
+                    System.out.println("Enter number of task: ");
+                    int markByNumber = scanner.nextInt();
+                    TaskManager.makeCompleted(markByNumber);
```

Comment:
```suggestion
                    TaskManager.markCompleted(markByNumber);
```
what's the `markByNumber`